### DISCUSSION TITLE
赤魔道師の魔法関連強化の要望

### DISCUSSION THREAD

**Discussion by Aquashrimp** [hengband/hengband#217] [Closed] [answered]
赤魔道師で使えない魔法(魔法剣など)を解禁してほしいのですが、どうでしょうか
ついでに魔法の全体的な習得Lvや難度も見直してもらいたいです

**Comment by deskull-m** [comment]
了解です。ひとまず私は肯定的に着手したいと思います。

**Comment by iks3** [comment, marked as answer]
Issues https://github.com/hengband/hengband/issues/319 にて対応しています。

以下の議論は以下のDiscussでお願いします。
[職業の調整案 #321](https://github.com/hengband/hengband/discussions/321)
